index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
I think we should make this a class method instead of a static method? Since builds are under performance-pressure, taking advantage of the natural caching of the loader in this way is useful.